week-1-practise/level-1/1-syntax-error-1.js

```diff
@@ -10,5 +10,8 @@ console.log(result); // Expected result 8 - Run the program and make sure it wor
 
 // Answer these questions:
 // 1. How many parameters does the function take?
+//3
 // 2. What is the function name?
-// 3. Where is the function called? with what parameters?
\ No newline at end of file
+//addNumbers
+// 3. Where is the function called? with what parameters?
+//console.log (results)
```

Comment:
When the question asks where the function is _called_, it means where was it executed (like this: `theFunction()`). It also asks what parameters it was given (what were the values of a, b, and c when it was called).
You have given the place where its return value is subsequently used, which is different.